scripts/makeupdates

```diff
@@ -409,7 +411,11 @@ def add_rpms(updates_path, rpms):
     for rpm in resolved_rpms:
         cmd = "cd %s && rpm2cpio %s | cpio -dium" % (updates_path, rpm)
         sys.stdout.write(cmd+"\n")
-        os.system(cmd)
+        ret = os.system(cmd)
+        print(f"ret code: {ret}")
+
+        if STRICT_ENABLED and ret != 0:
+            raise RuntimeError(f"RPM {rpm} can't be added to the updates image!")
```

Comment:
I still think it is wrong to add this and then hide behind a flag. If you want to unpack a rpm and it does not work, that's a problem. If that problem is silently skipped, that is a bug.

Reply:
Changed.